libs/ui/src/interfaces/activity.interface.ts

```diff
@@ -56,18 +80,30 @@ interface TransactionResponse {
 export interface ActivityResponse {
   cate_dict: unknown;
   history_list: TransactionResponse[];
-  project_dict: unknown;
+  project_dict: Record<string, ProjectInfo>;
   token_dict: Record<string, TokenInfo>;
 }
 
+interface Transfer {
+  amount: number;
+  symbol: string;
+}
+
 export interface ActivityData {
+  type: TransactionTypeEnum;
   hash: string;
   timestamp: number;
-  transactionLabel: TransactionLabelEnum;
-  transactionStatus: TransactionStatusEnum;
-  amount: number;
-  symbol: string;
+  label: string;
```

Comment:
this file starts to look like a mess, divide it into parts

Reply:
this file includes all activity types, I don't think we really need this

Reply:
@MuKoJla @Boltman92 what do you think?

Reply:
if we don't reuse this interfaces in another parts of app, only activity, I think it's ok to left all interfaces here

Reply:
I agree with Andrii, maybe it is not obvious how to spit, but it’s hard to read